packages/next/server/body-streams.ts

```diff
@@ -3,11 +3,14 @@ import { PassThrough, Readable } from 'stream'
 
 export function requestToBodyStream(
   context: { ReadableStream: typeof ReadableStream },
+  KUint8Array: typeof Uint8Array,
```

Comment:
**suggestion:** rename to `Uint8ArrayConstructor` or `Uint8ArrayConstr`

Took me a while to understand what that K was 😆